src/schema/v2/schema.ts

```diff
@@ -332,6 +333,7 @@ const rootFields = {
   shortcut,
   show: Show,
   showsConnection: Shows,
+  specialistBios: SpecialistBios,
```

Comment:
I don't think it should be a root field, should go under `staticContent` -

<img width="742" alt="Screen Shot 2024-04-09 at 10 16 47 AM" src="https://github.com/artsy/metaphysics/assets/1457859/e307e05c-e7d5-404a-896a-8d32c37f4599">

Reply:
Yes, thanks ☝🏽

Reply:
Oh, right. There is `staticContent` field under `root` already though and it does something different: https://github.com/artsy/metaphysics/blob/8239cc57c343db7a861d879ffdc09e5b3647e287/src/schema/v2/static_content.ts#L4

Should I create another namespace field? How would you call it? @anandaroop @mzikherman

Reply:
I was hoping we do not use it anywhere an can re-use it, but found queries like this in datadog:

```
staticContent(id: "how-auctions-work-bidding") {
    content(format: HTML)
    id
}
```

Reply:
I would probably use this field, make the `id` optional and resolve to `{}` when not passed in, and then add `specialistBios` under there.

A little odd, but I would rather use (and reuse, even if a bit odd) the `staticContent` field vs. root.

Reply:
Yeah I think that's^ reasonable, not much chance of colliding with the existing `content` and `id`, since the current and future staticContent additions will be properly scoped e.g. `staticContent.specialistBios.fooBar`

Reply:
makes sense 👍 solved